pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[build-system]
+# These are Pip's default requirements:
+# https://pip.pypa.io/en/stable/reference/pip/#pep-517-and-518-support
+requires = ["setuptools>=40.8.0", "wheel"]
+# Consider pinning YOUR build requirements more strictly.
```

Comment:
You really should remove this. That is not something we want to encourage and it is not the semantic intention of `requires`, so in an exemplar project like this we *really* don't want to cause a bunch of people to get stuck on old versions of `setuptools`.

Reply:
How about this instead?
> Beware that new backwards-incompatible releases may break the build.

Reply:
Please can you stop pushing this - both @pganssle and I have requested that we follow the "minimum requirements" approach, and leave pinning as something to discuss elsewhere. Please just remove the comment altogether.

Reply:
Thank you.

(BTW, are you force-pushing amended commits here? my comments seem to be getting detached from the code they were about).

Reply:
Sorry, you hadn't mentioned being against even warning the (probably new) user about the issue...

Reply:
> are you force-pushing amended commits here?

Ya, just `git commit -a --amend --no-edit && git push -f`... I think maybe it's a GitHub thing depending on the particular line a comment was left. Not sure, but maybe doing multiline comments would help?